library/src/main/kotlin/it/krzeminski/githubactions/actions/madhead/SemverUtils.kt

```diff
@@ -0,0 +1,21 @@
+package it.krzeminski.githubactions.actions.madhead
+
+import it.krzeminski.githubactions.actions.Action
+
+data class SemverUtils(
```

Comment:
```suggestion
data class SemverUtilsV1(
```
(see explanation in another comment)

Reply:
I cannot do that, because this Action doesn't have any versions, only the `latest` tag.